OracleDatabase/RAC/OracleConnectionManager/dockerfiles/21.3.0/cman.ora

```diff
@@ -0,0 +1,62 @@
+######################################################################
+#
+# Copyright (c) 2001,2002, Oracle Corporation. All rights reserved.
```

Comment:
Please change the copyright years to be accurate. 2001 is not the year this file was created and it was updated in 2022. It should also refer to "Oracle and/or its affiliates" and please remove "All rights reserved".